apps/web/src/components/feature-flag.tsx

```diff
@@ -0,0 +1,23 @@
+import { JSX, Show } from 'solid-js';
+
+export enum Feature {
+  none = 'none',
+  requests = 'requests',
+}
+
+type FeatureFlagProps = {
+  feature: Feature;
+  fallback?: JSX.Element;
+  children?: JSX.Element;
+};
+
+export function FeatureFlag(props: FeatureFlagProps) {
+  const features = JSON.parse(localStorage.getItem('features') ?? '{}');
+  console.log(features[props.feature]);
```

Comment:
**question/nitpick**

Is this still needed?

Reply:
I don't think it is, I rushed this part because my train was arriving haha

Reply:
We should add an eslint rule about this one :nerd_face:

Reply:
https://github.com/Bopzor/sel/pull/31/commits/a352e889cd6d36ddd0b05b7cbaa8269c26a19108#diff-0b67f0bda338ae0b05d8b4024b2e5915d659d1d191900217012909f8633a7ebeR40